README.md

```diff
@@ -97,29 +97,49 @@ tokio-stream = { version = "0.1" }
 near-lake-framework = "0.6.1"
 ```
 
-## Cost estimates
+## Cost estimates (Updated Mar 10, 2022 with more precise calculations)
```

Comment:
@khorolets we live in 2023, join us here 😄

Reply:
I also still need to travel to 2023 🤯